packages/bsky/tests/seeds/client.ts

```diff
@@ -383,3 +394,8 @@ export class SeedClient {
     return { authorization: `Bearer ${jwt}` }
   }
 }
+
+// @TODO use real admin auth
+const adminAuth = () => {
+  return ''
+}
```

Comment:
I'm curious whatsup with this little stub.  I think we _shooould_ have real admin auth working on the appview already.

Reply:
yeah it was just a broken import & I stubbed it out

meant to get back to it

Reply:
moved it to a param

we should probably move the whole seed client over to dev-env